src/streams.cpp

```diff
@@ -60,6 +65,7 @@ void AutoFile::ignore(size_t nSize)
             throw std::ios_base::failure(feof() ? "AutoFile::ignore: end of file" : "AutoFile::ignore: fread failed");
         }
         nSize -= nNow;
+        m_position += nNow;
     }
 }
```

Comment:
Out of scope but I found while reviewing:  Is there any reason why `AutoFile::ignore()` couldn't be removed and `AutoFile::seek(nSize, SEEK_CURR)` used instead?

Reply:
>  Is there any reason why AutoFile::ignore(nSize) couldn't be removed and AutoFile::seek(nSize, SEEK_CURR) used instead?

Keep in mind not all streams support seeking, so it would at the least be a conditional optimization. Agree it's not worth it as long as the amounts skipped are small (generally smaller or equal to the page size).